src/dms/frontend/EdFi.DataManagementService.Frontend.AspNetCore/appsettings.json

```diff
@@ -1,19 +1,19 @@
 {
     "AppSettings": {
-        "AuthenticationService": "http://localhost:5198/oauth/token",
+        "AuthenticationService": "http://localhost:8045/realms/edfi/protocol/openid-connect/token",
         "Datastore": "postgresql",
         "QueryHandler": "postgresql",
         "DeployDatabaseOnStartup": false,
         "BypassStringTypeCoercion": false,
-        "CorrelationIdHeader": "correlationid",
+        "CorrelationIdHeader": "",
         "AllowIdentityUpdateOverrides": "",
         "MaskRequestBodyInLogs": true,
         "UseLocalApiSchemaJson":  false
     },
     "IdentitySettings": {
         "EnforceAuthorization": true,
         "ServiceRole": "dms-client",
-        "Authority": "http://localhost:8045/realms/dms",
+        "Authority": "http://localhost:8045/realms/edfi",
```

Comment:
Our keycloak getting started document says to create a realm called "EdFi", not "DMS".